package.json

```diff
@@ -143,14 +141,17 @@
     "serve": "^12.0.1",
     "shebang-regex": "^2.0.0",
     "stream-to-string": "^1.2.0",
-    "ts-jest": "^24.2.0",
+    "ts-jest": "^28.0.8",
     "ts-node": "^7.0.1",
-    "typedoc": "^0.15.0",
-    "typescript": "^3.8.0",
+    "typedoc": "^0.23.10",
+    "typescript": "^4.0.0",
     "uuid": "^3.3.2",
     "web-help": "file:web-help",
     "yargs-parser": "^18.1.2"
   },
+  "overrides": {
+    "glob-parent": "~5.1.2"
+  },
   "engines": {
     "node": ">=8.0.0"
```

Comment:
Should we also update the engine to 16.0.0 ?

Reply:
It would probably be good to update it 🙂 Should we use ">=14.0.0" to include all versions of Node.js that are actively supported?

Reply:
yeah, makes sense to include 14 😋 
It should only throw warnings for people on 12 and below unless they have `engine-strict` set in their .npmrc configuration file